src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml

```diff
@@ -157,6 +157,6 @@ jms_serializer:
                 path: "@SyliusVariationBundle/Resources/config/serializer"
 
 a2lix_translation_form:
-    locale_provider: sylius.locale_provider
+    locale_provider: sylius.a2lix_locale_provider
```

Comment:
It's all right, `sylius.a2lix_locale_provider` uses `sylius.locale_provider` to get available locales. I divided previous `LocaleProvider` that implemented Sylius `LocaleProviderInterface` and A2lix `LocaleProviderInterface` into two classes.